Makefile

```diff
@@ -112,7 +112,7 @@ Client_C_Flags := $(SGX_COMMON_CFLAGS) -fPIC -Wno-attributes $(Client_Include_Pa
 
 Client_Rust_Path := target/$(OUTPUT_PATH)
 Client_Path := bin
-Client_Binary := substratee-client
+Client_Binary := integritee-cli
```

Comment:
Are you sure about `cli`? Client seems more adequate to me.

Reply:
Ok I see, I won't stand in the way then.

Reply:
client is used ambiguously by substrate, so I'd like to avoid the term